identity/src/main/java/life/qbic/identity/application/user/IdentityService.java

```diff
@@ -33,27 +32,15 @@
  *
  * @since 1.0.0
  */
-public final class UserRegistrationService {
+public final class IdentityService {
 
   private final UserRepository userRepository;
 
-  public UserRegistrationService(UserRepository userRepository) {
+  public IdentityService(UserRepository userRepository) {
     super();
     this.userRepository = userRepository;
   }
 
-  private static void sendNotification(DomainEvent event, NotificationService notificationService) {
-  Objects.requireNonNull(notificationService);
-    var notificationId = notificationService.newNotificationId();
-    var notification =
-        Notification.create(
-            event.getClass().getSimpleName(),
-            event.occurredOn(),
-            notificationId,
-            event);
-    notificationService.send(notification);
-  }
-
```

Comment:
I assume this functionality was moved somewhere? Can you point me in the right direction?

Reply:
it was removed since not used